src/js/angular/core/directives/queryeditor/query-editor.directive.js

```diff
@@ -85,6 +86,7 @@ function queryEditorDirective($timeout, $location, toastr, $repositories, Sparql
 
         $timeout(function () {
             drawQueryEditor(scope);
+            scope.queryEditorIsReady = true;
```

Comment:
How this guarantees that the editor would be ready?

Reply:
after "drawQueryEditor" the window.editor has value the YASQE reference with all overridden functions

Reply:
ahh yes, that freaking function ...